elasticsearch5/search-index/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java

```diff
@@ -400,13 +391,62 @@ void addElement(
             bulkUpdateService.flushUntilElementIdIsComplete(element.getId());
```

Comment:
We should wait in the BulkUpdateService when the batch is converted to a request.